Classes/ScrnPlayerController.uc

```diff
@@ -1487,6 +1487,33 @@ function ServerLockWeapons(bool bLock)
     bWeaponsLocked = Mut.bAllowWeaponLock && bLock;
 }
 
+//handles changing weapon's inventorygroup
+exec function MoveWepToSlot(int Slot)
```

Comment:
"Wep" is misspelled (Weap) and therefore confusing. I suggest a simpler name: **GunSlot**
`Slot` should be byte as it is in `SwitchWeapon()`

```suggestion
exec function GunSlot(byte Slot)
```